src/ReportPortal.Client/Service.cs

```diff
@@ -19,13 +19,14 @@ public partial class Service : IClientService, IDisposable
         /// <param name="projectName">A project to manage.</param>
         /// <param name="token">A token for user. Can be UID given from user's profile page.</param>
         /// <param name="httpClientFactory">Factory object to create an instance of <see cref="HttpClient"/>.</param>
-        public Service(Uri uri, string projectName, string token, IHttpClientFactory httpClientFactory = null)
+        /// <param name="ignoreSslErrors">Ignore SSL / TLS errors during https web requests</param>
+        public Service(Uri uri, string projectName, string token, IHttpClientFactory httpClientFactory = null, bool ignoreSslErrors = true)
```

Comment:
There is no need adding new parameter here. Ctor accepts `IHttpClientFactory` object which fully controls creation of `HttpClient`. There you can configure any http parameter you want. We cannot just add new ones here like *timeout* or *proxy*.